public/signin.html

```diff
@@ -1,4 +1,3 @@
-<!-->
```

Comment:
why there is an empty div at the end of the file?

Reply:
for completions reasons. note that this div is of class "col-sm-3". it's there to complete the other col-sm-x columns...

Reply:
@Evi1Pumpkin

Reply:
You dont have to complete the row to 12 columns.
You can leave it partial or use offsets/pull/push (look for it in the bootstrap documentation).